arch/sparc/Kconfig

```diff
@@ -32,6 +32,19 @@ config ARCH_CHIP_BM3823
 	---help---
 		Microchip BM3823 (ARCH_SPARC_V8)
 
+config ARCH_CHIP_S698PM
+	bool "S698PM"
+	select ARCH_SPARC_V8
+	select ARCH_HAVE_MATH_H
+	select ARCH_HAVE_IRQPRIO
+	select ARCH_VECNOTIRQ
+	select ARCH_HAVE_RAMFUNCS
+	select ARCH_HAVE_MULTICPU
+	select ARCH_HAVE_TESTSET
+	select ARCH_HAVE_SERIAL_TERMIOS
+	---help---
+		Microchip S698PM (ARCH_SPARC_V8)
```

Comment:
```suggestion
		ORBITA Sailing S698PM (ARCH_SPARC_V8)

Reply:
Using Microchip (to indicate chip) could make people think that Microchip company created it.